src/data/mv_polynomial/monad.lean

```diff
@@ -23,6 +23,12 @@ They interact in convenient ways with `mv_polynomial.rename`, `mv_polynomial.map
 Indeed, `mv_polynomial.rename` is the "map" operation for the (`bind₁`, `join₁`) pair,
 whereas `mv_polynomial.map` is the "map" operation for the other pair.
 
+## Implementation notes
+
+We add an `is_lawful_monad` instance for the (`bind₁`, `join₁`) pair.
+The second pair cannot be instantiated as a monad, since we do not have
+`mv_polynomial σ : Type → Type`; the type of coefficients must have a ring structure.
```

Comment:
```suggestion
The second pair cannot be instantiated as a `monad`,
since it is not a monad in `Type` but in `CommRing` (or rather `CommSemiRing`).
```